lib/blacklight/document/active_model_shim.rb

```diff
@@ -14,6 +14,11 @@ def primary_key
       def base_class
         self
       end
+
+      def find id
+        response = Blacklight.default_index.find id
+        self.new(response.documents.first, response)
```

Comment:
Ah, I see now.  Response is supposed to cast each document into blacklight_config.document_model.  I'll take the self.new out and clean up the tests below.